src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,25 @@
+
+
+package codeu.chat.common;
+import codeu.chat.util.Uuid; 
+
+public final class ServerInfo {
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Uuid version;
+  public Uuid temp = null;
+  //public final Uuid temp = Uuid.parse(SERVER_VERSION);
+  public ServerInfo() {
+  	try {
+  		temp = Uuid.parse(SERVER_VERSION);
+    	//this.version = Uuid.parse(SERVER_VERSION);
```

Comment:
This should be deleted since this isn't relevant anymore. It cleans up space and just deletes unnecessary lines.